modules/lvms-creating-logical-volume-manager-cluster.adoc

```diff
@@ -82,7 +83,7 @@ When configuring multiple device classes, you must specify the device path for e
 <2> Set `fstype` to `ext4` or `xfs`. By default, it is set to `xfs` if the setting is not specified.
 <3> Mandatory: The `LVMCluster` resource must contain a single default storage class. Set `default: false` for secondary device storage classes.
 If you are updating the `LVMCluster` resource from a previous version, you must specify a single default device class.
-<4> Optional. To control or restrict the volume group to your preferred devices, you can manually specify the local paths of the devices in the `deviceSelector` section of the `LVMCluster` YAML. The `paths` section refers to devices the `LVMCluster` adds, which means those paths must exist. The `optionalPaths` section refers to devices the `LVMCluster` might add. You must specify at least one of `paths` or `optionalPaths` when specifying the `deviceSelector` section. If you specify `paths`, it is not mandatory to specify `optionalPaths`. If you specify `optionalPaths`, it is not mandatory to specify `paths` but at least one optional path must be present on the node. If you do not specify any paths, then the `LVMCluster` adds the unused devices on the node. After a device is added to the `LVMCluster`, it cannot be removed.
+<4> Optional. To control or restrict the volume group to your preferred devices, you can manually specify the local paths of the devices in the `deviceSelector` section of the `LVMCluster` YAML. The `paths` section refers to devices the `LVMCluster` adds, which means those paths must exist. The `optionalPaths` section refers to devices the `LVMCluster` might add. You must specify at least one of `paths` or `optionalPaths` when specifying the `deviceSelector` section. If you specify `paths`, it is not mandatory to specify `optionalPaths`. If you specify `optionalPaths`, it is not mandatory to specify `paths` but at least one optional path must be present on the node. If you do not specify any paths, then the `LVMCluster` adds the unused devices on the node. You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. After a device is added to the `LVMCluster`, it cannot be removed.
```

Comment:
"Add" is a transitive verb so it should follow with an object.

How about:

> The `paths` section refers to devices added by the `LVMCluster`, so you must ensure the correct paths exist.

Reply:
Callout 4 is very large. Maybe create another 2 callouts that cover  `optionalPaths` and `paths` sections?

Reply:
I've been working on improving the existing documentation in a few sections. This proc is one of those.
This info was already present in the doc. The updates to this proc are part of this PR - https://github.com/openshift/openshift-docs/pull/70232